angular-ngrx-scss/src/app/auth/services/token.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
- [ ] Delete this test file please because this service is very basic, it does not need to be tested imho